src/liquid/components/ld-input/ld-input.tsx

```diff
@@ -133,24 +133,9 @@ export class LdInput implements InnerFocusable, ClonesAttributes {
   @State() clonedAttributes
 
   /**
-   * Emitted when the input looses focus.
+   * Emitted when the input value changed and the element loses focus.
    */
-  @Event() blur: EventEmitter<FocusEvent>
-
-  /**
-   * Emitted when the input value changes.
-   */
-  @Event() change: EventEmitter<string>
-
-  /**
-   * Emitted when the input receives focus.
-   */
-  @Event() focus: EventEmitter<FocusEvent>
-
-  /**
-   * Emitted when the input value changes.
-   */
-  @Event() input: EventEmitter<string>
+  @Event() ldchange: EventEmitter<string>
```

Comment:
Do I understand it correctly that this is an additional event to the `change` event, which is still available (though not documented anymore), except for when using React? If that is the case, we should probably document this somewhere.

Reply:
Yes, exactly. Maybe we should add a general section about native events to the docs, making clear that there's a difference between composed and non-composed native events and that we add additional custom events for some non-composed events. I would do that in a separate PR that we should also use to define a global convention for events in Liquid Oxygen and then refactor other components accordingly.